src/Flow-Core/CDBlockObjectVisitor.class.st

```diff
@@ -162,6 +141,27 @@ CDBlockObjectVisitor >> visitEnd: anEndState [
 "
 ]
 
+{ #category : #visiting }
+CDBlockObjectVisitor >> visitFork: aFork [ 
```

Comment:
As I said in the chat, a fork is a separation of control flow and decision a simple branch